test/cpp/rpc/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set(RPC_TEST_ROOT ${TORCH_ROOT}/test/cpp/rpc)
+
+file(GLOB RPC_TEST_SRCS ${RPC_TEST_ROOT}/test_*.cpp)
+set(RPC_TEST_SRCS ${RPC_TEST_SRCS} PARENT_SCOPE)
+
+add_executable(test_cpp_rpc
+  ${TORCH_ROOT}/test/cpp/common/main.cpp
+  ${RPC_TEST_ROOT}/gtest.cpp
+  ${RPC_TEST_SRCS})
+
+target_link_libraries(test_cpp_rpc PRIVATE torch gtest)
+target_include_directories(test_cpp_rpc PRIVATE ${ATen_CPU_INCLUDE})
+
+if (USE_CUDA)
+  target_link_libraries(test_cpp_rpc PRIVATE
+    ${CUDA_LIBRARIES}
+    ${CUDA_NVRTC_LIB}
+    ${CUDA_CUDA_LIB}
+    ${TORCH_CUDA_LIBRARIES})
```

Comment:
Isn't there a `torch_cuda` or equivalent target that pulls all of this in?

Reply:
Pieter,

I was using TARGETS/fb side to test, and trying to sprinkle parallel CMakeLists.txt incantations to attempt to make things work (and did a few re-uploads last night where I thought I got things to work). I was going back-and-forth about putting in the existing cpp/api/serialize.cpp test, and think I might just move the code back there.

Re: string format- I want to make sure I understand your question, but yes, the tensor data is contained in the emitted string from that function (as it was in torch::save()